screens/organizations/index.js

```diff
@@ -84,42 +79,42 @@ export function Organizations() {
               ["Volunteer", "https://faunalytics.org/volunteer"],
             ]}
           >
-            Faunalytics&apos; mission is to empower animal advocates with access
-            to research, analysis, strategies, and messages that maximize their
-            effectiveness to reduce animal suffering.
+            {t("organizations.faunalytics.description")}
           </Organization>
         </div>
 
         <div className="mx-auto max-w-xl space-y-14 text-center lg:max-w-2xl">
           <p className="leading-relaxed text-white">
-            We also highly recommend checking out{" "}
-            <a
-              href="https://www.wawcommittee.org/"
-              target="_blank"
-              className="underline"
-              rel="noreferrer"
-            >
-              Wild Animal Welfare Committee
-            </a>
-            ,{" "}
-            <a
-              href="https://longtermrisk.org/"
-              target="_blank"
-              className="underline"
-              rel="noreferrer"
-            >
-              Center on Long-Term Risk
-            </a>{" "}
-            and{" "}
-            <a
-              href="https://centerforreducingsuffering.org/"
-              target="_blank"
-              className="underline"
-              rel="noreferrer"
-            >
-              Center for Reducing Suffering
-            </a>{" "}
-            too!
+            <Trans t={t} i18nKey="organizations.description">
```

Comment:
Same as the other comments, but now the inner text has to be translated too. I'm not sure of the best way to do it here :smiling_face_with_tear:

Looks like translations can be reused inside other translations, so technically this should be possible
```json
  "organizations.description": "We also highly recommend checking out <2>$t('org1')</2>, <5>$t('org2')</5> and <9>$t('org3')</9> too!",
```

I might be going down the rabbit hole, but this looks possible too:

```json
"orgs": ["org1", "org2", "org3"],
 "organizations.description": "We also highly recommend checking out {{$t('orgs', list}} too!"
```

This would remove the formatting so it's automatically discarded but it can be pretty powerful :thinking:

Reply:
Will have a look at this later this week. I agree, right now it's pretty ugly 😅

Reply:
For now I want to leave it the way it is, but it's definitely something we can improve later on.